ocaml/database/master_connection.ml

```diff
@@ -215,6 +221,23 @@ let do_db_xml_rpc_persistent_with_reopen ~host:_ ~path (req : string) :
     else if !backoff_delay > 256.0 then
       backoff_delay := 256.0
   in
+  let rec delay = function
+    | d when d > 16.0 -> (
+        (* Sleep for a small time in case that the coordinator can connect but can't accept *)
+        delay (d /. 2.0) ;
+        match Atomic.get in_call_from_coordinator with
+        | true ->
+            debug
+              "%s: The coordinator is calling me. Wake up and retry connecting."
+              __FUNCTION__ ;
+            Atomic.set in_call_from_coordinator false
```

Comment:
This one doesn't seem needed, it's set to false in line 240 whenever the function delay finishes